### PR TITLE
Finish the categories

### PR DESCRIPTION
**Pull request by ArtemKech**
# CHANGES:
### Fixed: **_BBC_** & **_CNN_** spiders
*  _**BBC**_ - added categories to the spider. It fetches them all now.
*  _**CNN**_ - added categories to the spider. Fixed some of the fields that were not retrieving data. Time is in `UTC` format

Made spiders to fetch the categories that only exist in the `db`. I also merged some of the categories into big ones so that we would not end up having 100500+ different categories. 

Added some new categories into `db`. Let me know if you are happy about these new categories.

### EXTRA:
Increased the value for the following fields as requested:
* `name` from 128 to 256
* `author` from 128 to 256
* `publication_date`  from 24 to 32

---

### Closed issues: 
* ( #44 )
* ( #33 )
* ( #40 )

---
### **NOTE:** Do we want to keep `food` category?

### CODE REVIEW

**Comment by AardhynLavender** [comment]
@ArtemKech, Cant get scrappy to work from polytech today. I'll have a look this evening.

What categories are you using currently, I dont see the categories changed in the database.

File: database/src/init.sql [hunk 1]
```diff
@@ -26,10 +26,14 @@ create table Article (
 insert into Category (category, description)
 values  ('news', 'generic news articles'),
         ('gardening', 'gardening, landscaping, or botanical articles'),
-        ('motoring', 'cars or motor sports related articles'),
         ('politics', 'current political events and debate articles'),
         ('business', 'business, fintech, or economic articles'),
         ('culture', 'culturally significant articles'),
         ('world', 'world news and event articles'),
-        ('sport', 'generic sports articles');
```
Comment (AardhynLavender) [inline database/src/init.sql:34]: looks good.

File: scraper/newsscrapper/pipelines.py [hunk 2]
```diff
@@ -26,8 +26,8 @@ def serialize_item(item):
         articles = json.loads(data)
 
         # For checking purposes if it formats scraped data correctly
-        # with open('articles.json', 'w') as f:
-        #     json.dump(articles, f, indent=4)
+        with open('articles.json', 'w') as f:
+            json.dump(articles, f, indent=4)
```
Comment (AardhynLavender) [inline scraper/newsscrapper/pipelines.py:30]: Probably don't need this left in.

**Review by AardhynLavender — CHANGES_REQUESTED** [review]
Looking good!

Getting some key constraint errors on `article.create`

**Review by AardhynLavender — CHANGES_REQUESTED** [review]
Looking good, so close!

Couple of quick ones...

- Bumping `name` and `author` up to **varchar(256)` seems a better limit for those fields.
- Some `publication_date`s are legitimately larger than 24 characters. `varchar(32)` fits all our current timestamp instances.

I've noticed *Fox News* still has a few categories left to sort out--such as *realestate*. All the others are working great though 😁.

**Comment by ArtemKech** [comment]
@AardhynLavender. I made sure that the categories that are not in `db` not going to be scraped. Should work fine now

**Review by AardhynLavender — CHANGES_REQUESTED** [review]
Great stuff! 😎,

Can you make those quick alterations to the fields failing their length constraints.

Then its a thumbs up from me.

**Review by AardhynLavender — APPROVED** [review]
Thanks for that, 👍

**Pull request merged**